renderer/pages/index.tsx

```diff
@@ -157,7 +157,7 @@ export default function Home() {
       <Header />
 
       <main className="py-10">
-        <div className="grid max-w-3xl grid-cols-1 gap-6 mx-auto sm:px-6 lg:max-w-7xl lg:grid-cols-3">
+        <div className="grid max-w-3xl grid-cols-1 gap-6 mx-auto sm:px-6 lg:max-w-[1350px] lg:grid-cols-[31%_31%_34%]">
```

Comment:
I would set at least 1440px max width to see the difference between old one and use lg:grid-cols-[30%_30%_auto]. In this case you don't need to guess the width of manual input.